oap-server/server-storage-plugin/storage-zipkin-elasticsearch7-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/zipkin/ZipkinSpanRecord.java

```diff
@@ -120,7 +122,9 @@ public String id() {
             map.put(SPAN_ID, storageData.getSpanId());
             map.put(SERVICE_ID, storageData.getServiceId());
             map.put(SERVICE_INSTANCE_ID, storageData.getServiceInstanceId());
-            map.put(ENDPOINT_NAME, storageData.getEndpointName());
+            if (nonNull(storageData.getEndpointName())) {
+                map.put(ENDPOINT_NAME, storageData.getEndpointName());
+            }
```

Comment:
I think this should not be added. Once this has error, the code already has bug. Trace should not have no endpoint.